modules/custom/az_publication/config/install/core.entity_view_display.node.az_publication.az_card.yml

```diff
@@ -0,0 +1,161 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.az_card
+    - field.field.node.az_publication.field_az_accessed_date
+    - field.field.node.az_publication.field_az_authors
+    - field.field.node.az_publication.field_az_publication_abstract
+    - field.field.node.az_publication.field_az_publication_approximate
+    - field.field.node.az_publication.field_az_publication_category
+    - field.field.node.az_publication.field_az_publication_container
+    - field.field.node.az_publication.field_az_publication_date
+    - field.field.node.az_publication.field_az_publication_date_type
+    - field.field.node.az_publication.field_az_publication_doi
+    - field.field.node.az_publication.field_az_publication_extra_links
+    - field.field.node.az_publication.field_az_publication_image
+    - field.field.node.az_publication.field_az_publication_issue
+    - field.field.node.az_publication.field_az_publication_link
+    - field.field.node.az_publication.field_az_publication_location
+    - field.field.node.az_publication.field_az_publication_media
+    - field.field.node.az_publication.field_az_publication_page
+    - field.field.node.az_publication.field_az_publication_pmid
+    - field.field.node.az_publication.field_az_publication_publisher
+    - field.field.node.az_publication.field_az_publication_type
+    - field.field.node.az_publication.field_az_publication_version
+    - field.field.node.az_publication.field_az_publication_volume
+    - node.type.az_publication
+  module:
+    - field_group
+    - smart_title
+    - user
+third_party_settings:
+  field_group:
+    group_az_publication_reference:
+      children:
+        - az_publication_bibliography
+      label: Reference
+      parent_name: group_link
+      region: content
+      weight: 7
+      format_type: html_element
+      format_settings:
+        classes: 'text-muted font-weight-normal small mt-2'
```

Comment:
I think the `font-weight-normal` class may not be needed here. Also, personally I might prefer to remove the `small` class to keep the text at the normal size like our other cards, but I can understand that there may be reasons to have it at 14px instead of 16px.